NOTICES.md

```diff
@@ -22,4 +22,10 @@ Licensee’s name: Narbeh Shahnazarian
 
 Repository system identifier: narbs91
 
+---------------------------------------------------------------------------------
+
+Licensee’s name: lilypad-releases
+
+Repository system identifier: lilypad-releases
+
```

Comment:
I don't know that we need this for the release bot, but we can check with @bgins (and remove if unnecessary)